### ISSUE TITLE
DCH: low results

### ISSUE THREAD

**Issue by hbellafkir** [thulab/DeepHash#57]
I trained DCH on COCO using the default parameters in val_train script, which sets the iter_num to 2000. The results too low compared to the paper:

iter 33 prec    0.3673785145624646      rec     0.9312258941859837      mAP     0.4742411791076638
i2i_by_feature  0.5373119936770314
i2i_after_sign  0.48951766955994036
i2i_prec_radius_2       0.44337251780011877
i2i_recall_radius_2     0.6585969771414468
i2i_map_radius_2        0.518241127343017

Did i miss something the configuration file?

**Comment by fegler** [comment]
Did u find the missing configuration?
If you find, could you share your hyperparameter setting?
I also think there's something missing..
e.g., the default value of the quantization loss parameter(q_lambda) is set to 0.

**Comment by hbellafkir** [comment]
No i didn't. I'm still waiting for a answer.
Yes I noticed that too, q_lambda is set to 0 not only in DCH, but also in DTQ.

**Comment by sacralitus** [comment]
> No i didn't. I'm still waiting for a answer.
> Yes I noticed that too, q_lambda is set to 0 not only in DCH, but also in DTQ.

Hi! Could you help me please? I'm beginner in DL and have no idea how to test model I have trained. Model is somehow saved in .npy format. Don't know how to work in such a format therefore can't put bunch of my test images to test the CBIR model  :(
